index.bs

```diff
@@ -7065,7 +7065,7 @@ the compressor so it is comparable to the input level.
 </div>
 
 <div algorithm>
-	<dfn id="#envelope-rate">Computing the envelope rate</dfn> is done
```

Comment:
You can keep the ID on there if you want; leaving it off just means Bikeshed autogens one for you, but it's perfectly find to provide one manually instead.

Reply:
There seems to be only one place this is used, so I just left it. If there were more, a shorter ID would be useful.